app/furniture/marketplace/stripe_events_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
 
       when "checkout.session.completed"
         payment_intent = Stripe::PaymentIntent.retrieve(event.data.object.payment_intent, {api_key: marketplace.stripe_api_key})
-        order = marketplace.orders.find_by(id: payment_intent.transfer_group)
+        order = Order.unscoped.find_by(id: payment_intent.transfer_group, marketplace:)
```

Comment:
I don't love how this reads. It immediately makes me want to go and check why does this need to be unscoped and what does that mean. Does that mean that really this is a "cart_or_order"? Or should this be a "cart"? Or something else?

Reply:
This is a consequence of that default scope that I added; I'm going to revert that.